fsm.go

```diff
@@ -110,6 +110,9 @@ func (r *fsmS) lookupAgentHost(e *f.Event) {
 				r.scheduleRetry(e, r.lookupAgentHost)
 			})
 		}
+
+		r.agent.logger.Info("/proc/net/route does not exist. Scheduling retry.")
```

Comment:
Good catch @basti1302 👍 

Just a small thing here: I think we need to wrap it with `else` otherwise for procfs envs it will schedule retry twice. 
And what do you think if we change it from info to debug?

Reply:
Hm, I guess you are right about the `else`. I was under the impression that all other branches ended with a `return`, but there is indeed no `return` after the `go r.checkHost(gateway...` call. I'll add an `else` 👍 

I don't have a very strong opinion about the log level, but all other retries are announced at level `error`, so my thinking was that `info` is already quite a bit lower in severity. WDYT?

Actually, maybe I should also add the phrase `Cannot connect to the agent.` to the message. The non-existence of `/proc/net/route` is not that noteworthy but the fact that we could not yet connect to the agent is.

How about something like

```
r.agent.logger.Info("Cannot connect to the agent. Scheduling retry. (Connecting through the default gateway has not been attempted because proc/net/route does not exist.)")
```

If you rather had another level (debug, warn, error) just let me know.

Reply:
I've split the message into two messages now, one at level error in line with the other "Scheduling retry" messages and one at debug level. LMK if that works for you, or what you'd rather see there.

Reply:
Looks good to me! 👍

Reply:
Good to know. FWIW, your review status still says "requested changes" so merging is blocked.